test/test_pgfplotsx.jl

```diff
@@ -327,14 +327,14 @@ with(:pgfplotsx) do
     end
 
     @testset "Markers and Paths" begin
-        pl = plot(
+        p = plot(
```

Comment:
It's really hard to search for one letter variables

Reply:
Then we should unify all `... = plot(...)` calls to `pl = plot(...)` calls, will do.